### DISCUSSION TITLE
Use of open source

### DISCUSSION THREAD

**Discussion by ssddanbrown** [hargata/lubelog#516] [General]
Hello,
I saw you're advertising this application as open source so I'm just opening this to advise that this could potentially be misleading to many. The [customizations to the license](https://github.com/hargata/lubelog/blob/ddc3c2e1b5e18a40e1a992c5471cfd3b8a0b833f/LICENSE#L1-L4) puts limitations on use while open source, as per [the commonly regarded definition](https://opensource.org/osd), intends to ensure open use.

Just to be clear, I completely respect your right to license your efforts as you wish, I just believe that specifically marketing it as open source could be misleading, and I believe this kind of stretching the definition of open source can be harmful to term in general. Some alternative terms for this could be "Source Available" or "Fair Code".
If you're interested in why I think this is worth raising I've written further about this on my blog:

- https://danb.me/blog/why-open-source-term-is-important/
- https://danb.me/blog/open-source-available-distinction/

I think it can also be useful, if making modifications to your license, to not name your license the same as the existing license you've used as a basis.
When you've added extra terms like this it's no longer really the MIT license. Plus currently it can raise confusion as the MIT license alone provides wider rights as described in #505. It's not really being provided to individual/personal users under the MIT if the additional terms are expected to remain.

**Comment by hargata** [comment]
we're currently on hiatus, but when we return in November we will be switching to a different monetization model, which will include the removal of commercial restrictions.

Basically, LL will be licensed as MIT come November for all users, personal or commercial, but feature requests and whatnot will require a minimum of $10 donation before we will look at it and then a funding goal will be set anywhere from $100 to $100,000 before we start any development work on it, this shouldn't contradict with any MIT licensing terms. We will also be considering a sponsorship amount for a mention in the About section then, but this change will only take place when we return from our hiatus on November 1st. 

thanks.

**Reply by ssddanbrown** [reply]
Thanks for the response @hargata, and good to hear this will be resolved in the future.
Best of luck with the new funding model!
